node/openrpc.go

```diff
@@ -329,7 +369,8 @@ var blockNumberOrHashD = fmt.Sprintf(`{
           ]
         }`, blockNumberD, commonHashD, requireCanonicalD)
 
-var rpcSubscriptionParamsNameD = `{
+var rpcEthSubscriptionParamsNameD = `{
+		"title": "subscriptionName",
```

Comment:
@meowsbits added this, as it wasn't working for params without it even for `eth_subscribe`.